\section{Microarrays}
\label{microarray-discussion}

% This is the \emph{microarrays} section of the discussion.

\newthought{Microarray technology,} through the simultaneous assessment of the
expression of thousands of genes, became the first molecular biology tool
capable of addressing the complex polygenic nature of
cancer.\cite{grant_microarrays_2004} Genomic perturbations drive cancer
progression by disturbing mechanisms for cell cycle control, differentiation,
\smallcaps{dna} repair, apoptosis, tumour vascularization, and metabolism.  The
monitoring of gene expression signatures---as surrogates of biological
processes---in molecular profiles of cancer biopsies, can be used to investigate
how these mechanisms are impacted during cancer progression.

% A common approach to investigate how these mechanisms are impacted during cancer
% progression is through the monitoring of gene expression signatures, as
% surrogates of biological processes, in molecular profiles of cancer biopsies.

In this dissertation, we present the result of two analyses relating to the use
of gene expression signatures as biomarkers in cancer research.  The first
concerns the use of differentiation and proliferation signatures in cancer
diagnostic; the second regards the extent of prognostic signals in cancer
transcriptomes.  Here we offer a discussion of these contributions within the
broader context of the use of microarray technology in clinical oncology.  This
section will then conclude with some remarks on the challenges in the analysis
and interpretation of microarray data.

\subsection{Differentiation and proliferation signatures in cancer diagnostic}
\label{discussion-differentiation-microarrays}
% Monday 02Mar2015
Molecular classification of cancer is a common diagnostic problem in clinical
oncology.\cite{golub_molecular_1999,alizadeh_distinct_2000,bullinger_use_2004}
The problem consists in assigning tumours to known taxonomic classes based on
their expression profiles and is framed as a supervised learning prediction
task.  The conventional approach involves training a moleclular classifier in a
group of labeled samples and then assess its performance on an independent set
of unlabeled samples.\cite{golub_molecular_1999}

% This approach rests on the assumption that the core molecular features whose
% expression is necessary and sufficient to specify tumour classes are tractable
% by microarray technology.

This approach rests on the assumption that the core molecular features that
specify tumour classes are tractable by direct comparison of their expression
profiles.  While this is an established evidence in some cancer
models,\cite{haibe-kains_three-gene_2012,markert_molecular_2011} in other case
studies, molecular diagnostics of clinical sub-types is less
consensual.\cite{travis_new_2013,nikiforov_molecular_2011} This may be explained
in part by technical variance, e.g., data overfitting or different investigators
using different experimental methodologies.\cite{weigelt_challenges_2012}
Additionally, biological variance, in the form of noise due to sampling
heterogeneity or erratic patterns of tumour evolution, may also condition the
stability of classifiers derived from inter-class comparisons.

We sought to approach this classification problem from a different perspective.
Instead of relying on the \emph{intrinsic}, \emph{variant} features that appear
contrasted between samples of different classes, we addressed the task by
enrolling the \emph{extrinsic}, \emph{invariant} features of biomarkers for two
core processes of multicellular life: differentiation and proliferation.  As
cancer progression is defined by an increase in proliferation rates and a
concomitant decrease in tissue differentiation (Figure~\ref{fig:diff-prolif}),
we reasoned that mapping the expression profiles of distinct tumour sub-types
along these two continuums would result in a classification procedure that is
more resistant to technical and biological idiosyncrasies.

\begin{marginfigure}%
  \begin{center}
    % \includegraphics[width=9cm]{microarrays-economist.jpg}
    \includegraphics{proliferation-differentiation.png}
    \caption[Differentiation and proliferation in cancer]{A schematic
      representation of the inverse relationship between tissue differentiation
      and proliferation in cancer progression (see text for details).}
    \label{fig:diff-prolif}%
  \end{center}
\end{marginfigure}%

To test this idea, two requirements had to be met.  First, we needed a cancer
model characterized by a well defined linear progression, from benign,
differentiated tumour types, to aggressive, anaplastic ones---along which
taxonomic tumour classes could be sensibly represented.  Second, we needed a
robust method to define molecular differentiation signatures from expression
profiles of healthy tissues, and a dependable proliferation metagene.

As a case study, we took to thyroid cancer.  Thyrocyte-derived carcinomas are
broadly divided into well-differentiated, poorly differentiated and
undifferentiated types on the basis of histological and clinical parameters
(Figure~\ref{fig:thyroid-carcinogenesis}).\cite{kondo_pathogenetic_2006} Among
the well differentiated thyroid carcinomas are the papillary and follicular
types.  The anaplastic thyroid carcinoma, at the other extreme of the
dedifferentiation continuum, is a highly aggressive and lethal tumour
(Figure~\ref{fig:neoplastic-grading}).

\begin{marginfigure}
  \begin{center}
    \includegraphics{neoplastic-grading.png}
    \caption[Neoplastic grading]{In clinical pathology, the loss of tissue
      differentiation and increase in proliferation is captured by the concept
      of neoplastic grading.  While cancers with fair prognosis are said to be
      differentiated, cancers with poor prognosis are referred to as anaplastic.
      \textbf{A:}~Micrograph of a low magnification thyroid tissue.  The
      functional units of the thyroid gland are the thyroid follicles, lined by
      an epithelium of thyrocytes.  Thyrocytes delimit the follicular lumen,
      where the colloid serves as a reservoir for thyroglobulin.
      \textbf{B:}~Micrograph of an anaplastic thyroid carcinoma, a stage
      \smallcaps{iv} thyroid tumour.  These tumours have a high mitotic rate and
      are among the human tumours with the poorest prognosis.  Notice the degree
      of structural tissular disorganization compared with the tissue of
      origin.}
    \label{fig:neoplastic-grading}%
  \end{center}
\end{marginfigure}

Especially fitting to test our classification procedure is the distinction
between follicular adenomas and follicular carcinomas; and the distinction
between follicular variants of papillary carcinomas and their classical
counterpart.  These challenging pathological
diagnostics\cite{lubitz_molecular_2005} are critical from the prognostic point
of view.  In each case, while the former types behave in an indolent manner and
have an good prognosis, the latter are defined as poorly differentiated thyroid
carcinomas, and may evolve to develop a malignant phenotype.

Several methods exist to quantitatively measure cell proliferation in biological
samples, such as bromodeoxyuridine incorporation, Ki-67 or proliferating cell
nuclear antigen (\smallcaps{pcna}) immunostaining.  Conversely, the
differentiation state of a cell is commonly defined by a range of qualitative
morphological and physiological parameters.  Underlying these phenotypic traits,
at the molecular level, are tissue specific expression patterns that define the
degree of structural and functional specialization of their cellular types.

To investigate these expression patterns, we devised an agnostic method to
select for genes that are consistently highly expressed on a cell type of
choice, but among the least expressed in other tissue types.  This was
formulated by selecting for genes among the \num{1000} most expressed in the
tissue of choice, that were not among the top \num{5000} most expressed in an
assortment of other tissue types.  This algorithm was applied to a dataset of
sixteen \smallcaps{rna} sequencing profiles of healthy human
organs.\cite{bodymap_2.0_2012} Compared to microarray expression profiling,
\smallcaps{rna} sequencing technology estimates m\smallcaps{rna} expression with
read counts normalized by transcript length, therefore reflecting more
accurately absolute transcription levels.\cite{wang_rna-seq:_2009} This simple
learning algorithm yielded a list of eight thyrocyte specific genes
(Table~\ref{tab:diff-sigs}).\footnote{The list includes the genes
  \smallcaps{\emph{crabp1}}, \smallcaps{\emph{foxe1}}, \smallcaps{\emph{iyd}},
  \smallcaps{\emph{pth}}, \smallcaps{\emph{slc26a7}}, \smallcaps{\emph{tg}},
  \smallcaps{\emph{tpo}}, and \smallcaps{\emph{tshr}}.}

% four of which had previously been identified as
% canonical thyroid genes.\footnote{The list includes the genes
%   \smallcaps{\emph{crabp1}}, \smallcaps{\emph{foxe1}}, \smallcaps{\emph{iyd}},
%   \smallcaps{\emph{pth}}, \smallcaps{\emph{slc26a7}}, \smallcaps{\emph{tg}},
%   \smallcaps{\emph{tpo}}, and \smallcaps{\emph{tshr}}.  Four are known thyrocyte
%   genes: forkhead box protein \smallcaps{e1} (\smallcaps{\,\emph{foxe1}},
%   a.k.a.\,\smallcaps{\emph{ttf2}}), thyroglobulin, thyroperoxydase and the
%   \smallcaps{tsh} receptor.}

The thyroid differentiation biomarker was then projected in the feature space of
two \emph{Affymetrix} microarray platforms, \smallcaps{U}95av2 and
\smallcaps{U}133v2.  Because \emph{Affymetrix} platforms often bear several
probesets targeting for a specific gene, we selected, in two reference
compendiums of healthy tissues profiled with each platform, the probeset that
maximizes the thyroid-specific signal for each gene in our signature.  A thyroid
differentiation index, dubbed \emph{\mbox{t-index}}, could then be derived from
biological samples profiled in any of these platforms, by computing the median
expression of the respective selected probesets.

\begin{table}[ht]
  \small
  \centering
  % \fontfamily{ppl}\selectfont
  % \newcolumntype{d}[1]{D{.}{\cdot}{#1}}
  \begin{tabular}{lD{.}{.}{-1}}
    \toprule
    Human  BodyMap 2.0 tissue & \multicolumn{1}{c}{Number of tissue-specific genes} \\
    \midrule
    adipocytes                & 0                                                   \\
    adrenal gland             & 0                                                   \\
    blood                     & 19                                                  \\
    brain                     & 96                                                  \\
    breast                    & 5                                                   \\
    colon                     & 5                                                   \\
    heart                     & 13                                                  \\
    kidney                    & 18                                                  \\
    liver                     & 101                                                 \\
    lung                      & 14                                                  \\
    lymph nodes               & 0                                                   \\
    ovary                     & 5                                                   \\
    prostate                  & 5                                                   \\
    skeletal                  & 11                                                  \\
    testes                    & 68                                                  \\
    thyroid                   & 8                                                   \\
    \bottomrule
  \end{tabular}
  \caption[Size of tissue differentiation signatures]{Size of tissue
    differentiation signatures.  Tissue-specific differentiation signatures were
    derived by selecting for genes that are among the most expressed in the tissue
    of choice, and among the least expressed in the remaining 15 tissue types (see
    text for details).  The size of the signatures reflects the degree of
    structural and functional specialization of that organ.  Tissues for which no
    gene met the selection criteria are likely to have a less particular
    metabolism (adipocytes) or to represent a mix of different cell types (adrenal
    gland and lymph nodes).}
  \label{tab:diff-sigs}
\end{table}

A biomarker of proliferation was similarly derived from expression profiles of
healthy tissues.  The proliferating cell nuclear antigen (\smallcaps{pcna}) is a
cofactor of \smallcaps{dna} polymerase $\delta$ and an essential motif for cell
replication.  A metagene, called meta-\smallcaps{pcna}, was obtained by
selecting the 1\% genes most positively correlated with the
\smallcaps{\emph{pcna}} gene in a compendium of expression profiles of normal
tissues.\cite{venet_most_2011} This metagene consists of 129 genes featuring
many significant cell-cycle related genes, like \smallcaps{\emph{aurka}},
\smallcaps{\emph{mki67}}, \smallcaps{\emph{top2a}}, or \smallcaps{\emph{mcm2}}.
A meta-\smallcaps{pcna} index can similarly be derived from expression profiles
of biological samples by computing the median expression of the genes of this
proliferation biomarker.

% We then tested the expression of these two biomarkers in three datasets of
% normal and neoplastic thyroid expression profiles.  The first, hybridized on an
% \emph{Affymetrix} \smallcaps{u}133v2 chip, comprised eleven anaplastic thyroid
% carcinomas (\smallcaps{atc}s), together with \num{49} papillary thyroid
% carcinomas (\smallcaps{ptc}s), paired with \num{45} adjacent normal tissues
% (\smallcaps{gse}29265).  The second included seven normal thyroid samples
% (\smallcaps{n}), nine follicular thyroid carcinomas (\smallcaps{ftc}s); and an
% aditional 17 follicular thyroid adenomas (\smallcaps{fta}s), 9 \smallcaps{ftc}s,
% 13 follicular variants of papillary thyroid carcinomas (\smallcaps{fvptc}s) and
% nine classical papillary thyroid carcinomas (\smallcaps{cptc}s)---altogether
% hybridized on \emph{Affymetrix} \smallcaps{U}95av2 chips and normalized with the
% \smallcaps{rma} procedure (\smallcaps{gse}29315).

We then evaluated the expression of these two biomarkers in three datasets of
normal and neoplastic thyroid expression profiles.  The first, hybridized on an
\emph{Affymetrix} \smallcaps{u}133v2 chip
(\href{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29265}{\smallcaps{gse}29265}),
comprised a selection of 49 samples, including anaplastic thyroid carcinomas
(\smallcaps{atc}s) and papillary thyroid carcinomas (\smallcaps{ptc}s), paired
with their respective adjacent normal tissues.  The second, hybridized on
\emph{Affymetrix} \smallcaps{U}95av2
(\href{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29315}{\smallcaps{gse}29315}),
included a total of 71 samples spanning normal thyroids, follicular thyroid
adenomas and follicular thyroid carcinomas (\smallcaps{fta}s and
\smallcaps{ftc}s); altogether with classical papillary thyroid carcinomas and
follicular variants of papillary thyroid carcinomas (\smallcaps{cptc}s and
\smallcaps{fvptc}s).  The third dataset\cite{van_staveren_gene_2006} comes from
a kinetic time course study profiling primary cultured thyrocytes with
\mbox{thyroid-stimulating hormone} (\smallcaps{tsh}), and was hybridized on a
home made platform interrogating nearly \num{4000} genes.  A well characterized
response of thyrocytes in culture to \smallcaps{tsh} stimulation is an increase
both in metabolic activity (differentiation), as well as in mitotic activity
(proliferation).

By quantifying the \emph{t-index} and the meta-\smallcaps{pcna} index in these
three datasets, we were able to establish that, (\emph{a}) the two indices are
negatively correlated in a range of thyrocyte-derived tumours of increasing
aggressiveness, yet positively correlated in a time course experiment of
\smallcaps{tsh} stimulation of thyrocytes; (\emph{b}) the \emph{t-index} can
accurately discriminate between expression profiles of \smallcaps{fta}s when
compared with \smallcaps{ftc}s; and between expression profiles of
\smallcaps{fvptc}s when compared with \smallcaps{cptc}s; and (\emph{c}) the
performance of this differential diagnosis classifier is as robust as a
classifier derived by training a \smallcaps{svm} learning algorithm (validated
with a repeated inner/outer cross-validation procedure) on the whole expression
space of the labeled samples.

\medskip

Because defects in cell-cycle regulation are the defining feature of neoplastic
pathogenesis, genes participating in proliferation are often found highly
expressed in tumour microarrays when compared with normal samples.  In spite of
the many discordant proliferation gene lists proposed in the
literature,\cite{whitfield_common_2006} increased expression of most of these
biomarkers has often been linked with poor clinical
prognosis.\cite{dai_cell_2005,paik_multigene_2004,rosenwald_proliferation_2003,sorlie_gene_2001}
Proliferation is a universal and conserved theme of cancer
transcriptomes,\cite{rhodes_large-scale_2004} and frequently accounts for most
of the power driving the performance of prognostic
signatures.\cite{sole_biological_2009,venet_most_2011,wirapati_meta-analysis_2008}
Proliferation biomarkers are thus a major component of genomic-based clinical
diagnostics for cancer patients.

The identification of other potential tumour class-specific genes, related to
the particular biology of each taxonomic group, is performed by training
learning algorithms on labeled expression profiles.  The validity of the
selected features, along with the mathematical function used to predict tumour
class based on their vector of expression, is then assessed by testing the
accuracy of the model in unlabeled samples.\cite{simon_diagnostic_2003} This
methodology can however be hindered by a number of
issues.\cite{brenton_molecular_2005} The classifier may reflect the inherent
technical and biological biases specific to the training set, rather than
capturing the modulations underpinning class specification---the model is then
said to overfit the training set.  Moreover, even when models are trained in
large enough datasets and proper care is taken to ensure independent validation,
the unstable nature of cancer genomes may itself obscure class-specific
biological signals by adding random variance to expression measurements.

Here, we investigate the possibility of enlisting another classifier, external
to cancer biology, to address the problem of thyrocyte-derived tumour class
prediction.  By identifying genes whose expression rank highly exclusively in
thyroid compared to other tissues, we defined a molecular differentiation marker
that takes no \emph{a priori} concerning the biology of the thyrocyte.
Unsurprisingly, this marker contains important genes in thyroid physiology,
including genes coding for: thyroglobulin (a dimeric protein used to produce
thyroid hormone); thyroid peroxidase (a membrane-bound glycoprotein that takes
part in the iodination of the precursors of thyroid hormone); iodotyrosine
deiodinase (an enzyme that facilitates the iodide salvage post-thyroid hormone
synthesis); the thyrotropin receptor (a receptor activated by \smallcaps{tsh});
and the \smallcaps{ttf2} transcription factor (active in the developing thyroid,
with a role in controlling the onset of its
differentiation\cite{zannini_ttf-2_1997}).  The signature also comprises genes
coding for the parathyroid hormone (parathyroid cells were likely to be present
in the resected profiled biopsies); cellular retinoic acid binding protein 1 and
a member of the solute carrier family 26 (both proteins for which no known role
in thyroid has been identified to date).

Noticeably, in diagnostic immunochemistry, antigens for thyroglobulin and
thyroid peroxidase are already routinely used to provide a quantitative
assessment of the malignancy of neoplastic thyroid
lesions.\cite{tanaka_immunohistochemical_1996,gerard_correlation_2003} Our
biomarker of thyrocyte differentiation, the \emph{t-index}, was used as a
classifier to accurately disentangle two challenging pathological diagnoses,
\mbox{\smallcaps{fta} \emph{vs} \smallcaps{ftc}} and \mbox{\smallcaps{fvptc}
  \emph{vs} \smallcaps{cptc}}, based on their expression profiles
(\mbox{\emph{cf.}  Figure~4\emph{a},\emph{d}} of the article in the
\hyperref[chap:results]{\textsf{Results}} chapter).  The \emph{t-index} can also
be used to chart a linear progression between thyroid neoplasias of increasing
malignancy (\mbox{\emph{cf.}  Figure~2\emph{a},\emph{b}} of the article in the
\hyperref[chap:results]{\textsf{Results}} chapter).  Papillary and follicular
carcinomas, the two main taxonomic classes of thyroid tumours, are shown to
depart from thyrocytes along two distinct axes of gene expression
(\mbox{\emph{cf.} Figure~3} of the article in the
\hyperref[chap:results]{\textsf{Results}} chapter), suggesting that our
biomarker correlates with the molecular courses of thyroid cancer progression.

Furthermore, we show that the \emph{t-index} and the meta-\smallcaps{pcna} index
are negatively correlated in expression profiles of a panel of thyroid cancers
(\mbox{\emph{cf.} Figure~2\emph{a},\emph{b}} of the article in the
\hyperref[chap:results]{\textsf{Results}} chapter), yet positively correlated in
an \emph{in vitro} experiment of physiological and mitotic thyrocyte activation
(\mbox{\emph{cf.} Figure~2\emph{c}} of the article in the
\hyperref[chap:results]{\textsf{Results}} chapter).  This provides supporting
evidence for the independence of our classifier from proliferation, unlike
potential classifiers derived from the expression space of labeled tumour
samples.  Nonetheless, class prediction of thyroid tumours based on the
\emph{t-index} is as accurate as a state-of-the art machine learning algorithm
selecting for the optimal classifying genes from the entire set of features
spotted in the arrays (\mbox{\emph{cf.} Figure~4\emph{c},\emph{f}} of the
article in the \hyperref[chap:results]{\textsf{Results}} chapter).

This proof of concept brings into a quantitative framework the formulation that
neoplastic progression can be mapped along an axis of molecular
dedifferentiation.  Thyroid cancer is especially suited to test this hypothesis,
as it comprises a family of mostly indolent tumours that evolve slowly
(Figure~\ref{fig:thyroid-carcinogenesis}), providing ample opportunity for
sampling malignancies at different stages of progression.  In addition, biopsies
of thyrocyte-derived cancers, while populated by a range of distinct cellular
types (fibroblasts, \smallcaps{c} cells, stromal cells), are mostly dominated by
thyroid epithelial cells---an essential condition for the biological bearing of
our differentiation signature in these expression profiles.

The generalization of this approach to other cancer families may depend on both
the feasibility to derive stable markers of differentiation for the cell type of
origin, as well as on the tractability of those markers in respective cancer
biopsies.  Breast cancer, for instance, is typically derived from epithelial
cells of the mammary gland, yet the expression profiles of most breast cancer
biopsies are largely dominated by adipocytes.  On the other hand, cancers
derived from the hematopoietic lineage could prove particularly amenable to our
strategy, as expression profiles of purified populations of human hematopoietic
cells at different stages of maturation are readily available in the
literature.\cite{novershtern_densely_2011}

While the use of biologically motivated molecular markers to dissect cancer
genomes is not a novelty,\cite{sund_tumor_2009,dave_prediction_2004} to our
knowledge, ours is the first study to enroll biomarkers of differentiation to
guide the interpretation of cancer expression profiles.  We devised a simple
method to catalogue genes that are only significantly expressed in a particular
tissue type, potentially seizing fundamental tissue-specific expression
patterns.  We then showed that a thyroid-specific biomarker can be reliably used
to map the molecular progression of two distinct taxonomies of thyrocyte-derived
neoplasias, and is able to solve two challenging pathological diagnoses.  These
results raise the possibility that the process of molecular dedifferentiation, a
collateral of cancer progression, could be accurately quantified and modeled for
prospective use in clinical oncology.

Our methodology is not without shortcomings.  Not all genes in the thyroid
differentiation signature are pertinent to the class prediction problem.  For
instance, the expression of the \smallcaps{tsh} receptor gene is usually
preserved during cancer progression,\cite{dagostino_different_2014} and the
expression of the parathyroid hormone gene is irrelevant to the thyrocyte's
biology.  More sophisticated learning methods, along with more stringent
techniques of cell type isolation prior to profiling, might help to alleviate
these imprecisions.  Additionally, for each of the undertaken classification
tasks, we do not provide a proper cut-off point to translate the quantitative
predictive \emph{t-index} into a predicted class label.  This is because our
experimental setting did not include a sufficient number of samples of each
class to estimate a sensible cut-off point for the classifier.  In both cases,
the accuracy of the classifier was estimated by stipulating all possible cut-off
points and computing the respective area under the \smallcaps{roc} curve
(\mbox{\emph{cf.}  Figure~4\emph{b},\emph{e}} of the article in the
\hyperref[chap:results]{\textsf{Results}} chapter).

Still, when addressing tumour class prediction, classifiers based on
differentiation signatures have the advantage of being stable and extraneous to
the biology of the disease, and thus virtually immune to biological variance
motivated by cancer genome instability or overfitting biases.  Their potential
to assist diagnostic tasks in cancer oncology remains to be fully explored.

% is marked by the incremental shedding of functional
% features specifying cell differentiation.  It raises the intriguing possibility

\subsection{The extent of prognostic signals in the cancer transcriptomes}
\label{sec:discussion-prognostic-microarrays}
% Tuesday 03Mar2015

\begin{marginfigure}%
  \includegraphics{nem-vandevijver-2002-fig2ab.pdf}
  \caption[Validation of a genomic marker's predictive ability]{Validation of a
    genomic marker's predictive ability.  A predictive 70-gene signature was
    derived from a prospective cohort of 98 expression profiles of breast cancer
    with known survival times (\citealp{vant_veer_gene_2002}).  This
    prognosis-classifier was subsequently used to segregate good and bad
    prognosis groups in an independent cohort of 295 breast
    carcinomas. Differential outcome between each group, regarding likelihood of
    developing metastasis (panel \textbf{A}), or likelihood of dying (panel
    \textbf{B}), was then established with Kaplan-Meier analysis
    (\citealp{van_de_vijver_gene-expression_2002}).}
  \label{fig:validation-prognosis-km}%
\end{marginfigure}

Disease outcome prediction is another important challenge in clinical
oncology.\cite{van_de_vijver_gene-expression_2002,vasselli_predicting_2003,sanchez-carbayo_defining_2006}
The problem consists in uncovering biomarkers whose expression in cancer
biopsies have a predictive ability regarding disease outcome.  It can also be
conceived as a supervised learning prediction task, with a similar strategy to
the class prediction problem.  A molecular classifier is derived from a training
set of samples with distinct survival times with respect to a clinical event;
its predictive ability is then validated with survival analysis on an
independent set of samples with associated clinical follow-up data
(Figure~\ref{fig:validation-prognosis-km}).

This methodology has identified cancer prognostic and predictive signatures with
superior performance to conventional histopathological or clinical
parameters.\cite{sole_biological_2009} Additionally, it offers a framework to
test for the implication of particular biological processes in cancer
progression, through the quantification of the association of their surrogate
transcriptional markers with clinical outcome.\cite{chang_gene_2004} This
formulation assumes that the expression patterns prompting cancer progression
are universal motifs, yet specific enough to be modeled by biologically
motivated molecular markers.

We sought to test this assumption by characterizing the range and the nature of
prognostic transcriptional signals in a representative selection of human cancer
expression profiles.  In order to do so, we compiled 114 expression profile
studies of cancers afflicting 19 organ systems (Table~\ref{tab:cancer-tissues}),
with clinical follow-up data.  The transcriptomes of these nearly \num{22000}
neoplastic samples were profiled with over 30 different commercial and custom
microarray platforms (Table~\ref{tab:datasets}).

\begin{table}[ht]
  \small
  \centering
  % \fontfamily{ppl}\selectfont
  % \newcolumntype{d}[1]{D{.}{\cdot}{#1}}
  \begin{tabular}{lD{.}{.}{-1}}
    \toprule
    \multicolumn{1}{c}{Tissue of origin}   & \multicolumn{1}{c}{Number of cancer studies} \\
    \midrule
    breast                                 & 33                                           \\
    blood cells                            & 12                                           \\
    central nervous system                 & 11                                           \\
    colon                                  & 9                                            \\
    lung                                   & 8                                            \\
    ovary                                  & 8                                            \\
    bladder                                & 6                                            \\
    lymphatic system                       & 5                                            \\
    kidney                                 & 3                                            \\
    liver                                  & 3                                            \\
    prostate                               & 3                                            \\
    uterus                                 & 3                                            \\
    bone                                   & 2                                            \\
    % cervix                               & 2                                            \\
    squamous cells (mouth, nose or throat) & 2                                            \\
    skin                                   & 2                                            \\
    adipocytes                             & 1                                            \\
    pancreas                               & 1                                            \\
    stomach                                & 1                                            \\
    thyroid                                & 1                                            \\
    \bottomrule
  \end{tabular}
  \caption[Distribution of tissues of origin in the cancers studies probed for
  prognostic signals]{Nearly \num{22000} expression profiles of cancer
    biopsies, across 114 studies, were compiled from the
    public domain to quantify the extent of prognostic signals in cancer
    transcriptomes.  Among these, experiments profiling breast tumours, with 33
    studies, were the most represented. This reflects the prevalence of breast
    cancer in the population (Figure~\ref{fig:globocan}), and its pivotal
    role as the original model for genomic outcome prediction analyses.  Next, in terms
    of representation, are studies profiling cancers originating from the
    hematopoietic lineage (12) and central nervous system (11)---perhaps a
    reflex of the ability to isolate relatively uncontaminated populations of tumoural cells from
    these cancer types---, followed by colon (9) and lung (8)---two of the most prevalent cancers
    worldwide.  See Table~\ref{tab:datasets} for a thorough description of each dataset used in
    this meta-analysis.}
  \label{tab:cancer-tissues}
\end{table}

To gauge the extent of prognostic signals in each of these datasets, we tested
association with outcome of each of the 4722 biologically motivated gene
expression signatures in the \smallcaps{c2} collection of the Molecular
Signatures Database (MSigDB \smallcaps{c2}), curated by the Broad Institute.
This collection includes gene sets collected from various sources such as online
pathway databases, publications in PubMed, and knowledge of domain experts.

Estimating association between a genomic marker and clinical outcome requires
the formulation of an outcome predictor from multi-gene values of expression.
Traditionally, this is achieved by a function that stratifies the cohort in good
and bad prognosis groups according to a cut-off point for the expression of the
gene classifier (Figure~\ref{fig:validation-prognosis-km}).  Instead, we chose
to formulate association with outcome by modeling survival time as a function of
the first principal component of each expression signature.  The logrank
\mbox{\emph{p-}value} of the corresponding Cox proportional model was used to
assert the prognostic value of the biomarker.

% \begin{table}[ht]
%   \small
%   \centering
%   % \fontfamily{ppl}\selectfont
%   % \newcolumntype{d}[1]{D{.}{\cdot}{#1}}
%   \begin{tabular}{lD{.}{.}{-1}}
%     \toprule
%     \multicolumn{1}{c}{Gene Ontology Biological Process term}                                     & \multicolumn{1}{c}{Frequency} \\
%     \midrule
%     innate immune response                                                                        & 39                            \\
%     neurotrophin \smallcaps{trk} receptor signaling pathway                                       & 33                            \\
%     positive regulation of transcription from \smallcapse{rna} polymerase \smallcaps{ii} promoter & 33                            \\
%     blood coagulation                                                                             & 32                            \\
%     negative regulation of apoptotic process                                                      & 31                            \\
%     Fc-epsilon receptor signaling pathway                                                         & 29                            \\
%     apoptotic process                                                                             & 27                            \\
%     epidermal growth factor receptor signaling pathway                                            & 27                            \\
%     fibroblast growth factor receptor signaling pathway                                           & 26                            \\
%     platelet activation                                                                           & 25                            \\
%     positive regulation of apoptotic process                                                      & 24                            \\
%     response to drug                                                                              & 24                            \\
%     mitotic cell cycle                                                                            & 21                            \\
%     viral process                                                                                 & 21                            \\
%     axon guidance                                                                                 & 19                            \\
%     Ras protein signal transduction                                                               & 18                            \\
%     negative regulation of cell proliferation                                                     & 18                            \\
%     negative regulation of transcription from \smallcaps{rna} polymerase \smallcaps{ii} promoter  & 18                            \\
%     positive regulation of cell proliferation                                                     & 18                            \\
%     signal transduction                                                                           & 18                            \\
%     \smallcaps{g1}/\smallcaps{s} transition of mitotic cell cycle                                 & 17                            \\
%     insulin receptor signaling pathway                                                            & 17                            \\
%     leukocyte migration                                                                           & 16                            \\
%     positive regulation of transcription, \smallcaps{dna}-templated                               & 16                            \\
%     protein phosphorylation                                                                       & 16                            \\
%     \bottomrule
%   \end{tabular}
%   \caption[Frequency of most represented \smallcaps{go} Biological Process terms
%   among the genes of 4722 gene sets of MSigDB \smallcaps{c2}]{Frequency of most
%     represented \smallcaps{go} Biological Process terms among the genes of 4722 gene sets of MSigDB \smallcaps{c2}.}
%   \label{tab:go-msigdb}
% \end{table}

To control for an over-representation of cancer-related themes within the
\mbox{MSigDB \smallcaps{c2}} collection, we replicated the analysis with a
synthetic collection of signatures of the same size, but made up of genes
randomly selected from the human genome.  Finally, we tested association of
single features in every dataset with outcome, by modeling survival times as a
function of individual vectors of expression in the expression matrices.

Figure~\ref{fig:prognostic-fraction} reports the proportion of tested
\mbox{MSigDB \smallcaps{c2}} signatures, randomized signatures, and single
features found associated with outcome in each dataset at a logrank $p<0.05$.
These quantities capture the fraction of each transcriptome bearing information
regarding differential patient survival.  From the statistical point of view,
they can be interpreted as likelihoods of finding a significant genomic
association with outcome by chance alone.  We thus refer to these estimates as
the baseline prognostic content\footnote{Prognostic content will henceforth
  refer to the fraction of tested markers found associated with outcome in a
  given dataset.} of cancer transcriptomes.

Critically, in 100 of the 114 datasets analyzed (88\%), more than five percent
of single-gene markers show a significant association with outcome.  When
considering multi-gene markers, up to 87 datasets (76\%) registered more than
five percent of random signatures associated with outcome.  Overall, the median
prognostic content across all studies for single-gene markers and random
multi-gene markers was, respectively, 12\% and 16\%.  These observations suggest
that, in most cancer transcriptomes, prognostic signals are disseminated
throughout a nontrivial fraction of their expression features; that the
likelihood of finding random associations with outcome is globally
non-negligible; and that the nature of prognostic signals is not
marker-specific.

We investigated the possibility of these estimates being the result of biased
survival time constructs.  To do so, we measured the prognostic content captured
by single-gene markers in the thirteen breast cancer datasets with overall
survival data available in our study
(Figure~\ref{fig:distribution-survival-times}), for \num{100} iterations each of
the respective randomized survival objects.  In ten out of the thirteen breast
datasets thusly tested, the actual observed prognostic content was beyond the
95\textsuperscript{th} quantile of the null distribution generated with
randomized survival objects (Figure~\ref{fig:null-breast-os}).  Furthermore, in
all but one of the datasets (\smallcaps{GSE9893}), the median of the null
distributions did not depart remarkably from the five percent stipulated
significance level.  This result suggests that the transcriptomic prognostic
signals quantified in this meta-analysis are conducive of actual biological cues
regarding the prospective malignancy of the profiled cancers.

A compelling result of this analysis is the considerable heterogeneity of
prognostic fractions observed across surveyed cohorts
(Figure~\ref{fig:prognostic-fraction}).  This observation is not
cancer-specific.  Among breast cancers, for instance, single-marker prognostic
content alone ranged from 4\% to 60\%.  To examine the contributions of
potential biological and demographic dataset-specific factors to this effect, we
quantified single-marker prognostic fraction for re-samplings of the \num{1972}
transcriptomes of the two \smallcaps{metabric} breast cancer cohorts, regarding
modulations of four variables: sample size, duration of follow-up time, fraction
of \smallcaps{er}+ patients, and fraction of node positive patients
(Figure~\ref{fig:bootstrap-metabric}).

The estimates of prognostic content are markedly sensitive to sampling variance,
as suggested by the dispersal of the distribution of estimates (confidence
interval: 12\% to 23\%), when \num{100} samplings of \num{500} random
transcriptomes were examined for the fraction of genes associated with outcome
(Figure~\ref{fig:bootstrap-metabric}\emph{a}).  This feature alone is likely to
yield a significant contribution to the range of estimates computed in our
meta-analysis, as 108 of the 114 datasets analyzed included less than \num{500}
profiled tumours.  Moreover, the sensitivity of our estimation procedure is, as
expected, largely dependent on the number of profiles included in the analysis,
as shown by a bootstrapping experiment of sample sizes towards the assessment of
the prognostic fraction (Figure~\ref{fig:bootstrap-metabric}\emph{b}).
Provocatively, the estimates of prognostic content do not appear to level off
even when the experimental sample size reached \num{1750}---by far the largest
in the field.  A sampling experiment of sequential truncation of follow-up times
was equally shown to impact estimates of prognostic content
(Figure~\ref{fig:bootstrap-metabric}\emph{c}).  Sharply increasing predictive
fractions were observed up to the fifth year of follow-up, followed by a gradual
decrease of the estimates for higher follow-up times.  This trend suggests that,
in breast cancers, prognostic patterns of expression are optimally correlated
with short-term forms of progression of the disease, and that long-term forms of
progression are less efficiently predicted from primary tumour transcriptomes.
The modulation of the fraction of \smallcaps{er}+ transcriptomes towards
experimental samplings of our estimate has exposed a tendency congruent with the
clinical relevance of this receptor in breast cancer pathology
(Figure~\ref{fig:bootstrap-metabric}\emph{d}).  Thus, an increase of
\smallcaps{er}+ profiles to up to 50\% in our samplings leads to a corresponding
linear rise in estimates of prognostic content, at which point a further
increase in the proportion of \smallcaps{er}+ profiles yields little impact on
fraction estimates.  This observation is in line with the fact that the
predictive power of most signatures in breast cancer is mostly confined to
\smallcaps{er}+ phenotypes.\cite{weigelt_challenges_2012} Finally, a sampling
experiment with dosed fractions of profiles from node positive patients
(Figure~\ref{fig:bootstrap-metabric}\emph{e}), has also uncovered a pattern of
prognostic fraction estimates that can be largely explained by the fact that
nodal status is clinically correlated to \smallcaps{er} status in breast cancer.

\medskip

A large number of studies has interpreted the statistical association of
transcriptional markers with clinical outcomes as evidence that their underlying
biological mechanisms are involved in cancer progression.  This approach has
gained widespread appeal because of the increasing availability in the public
domain of expression profiles of cancer biopsies with associated survival data;
and because it obviates the need for time-consuming and costly experimental
setups on \emph{in vivo} models.

In order for a molecular marker to be recognized as a prognostic factor, it has
to meet three successive criteria: its \emph{specific} association with clinical
outcomes has to be asserted; its prognostic performance and accuracy has to be
validated in an independent group of patients; and the independence of its
prognostic value from other potential factors has to be established with a
multivariate analysis.\cite{chibon_cancer_2013}

The assumption that prognostic signatures reported in the literature are
specific conveyors of biological signals pertaining to cancer progression was
previously challenged by our research group and
others.\cite{venet_most_2011,lauss_prediction_2010,ein-dor_outcome_2005,mosley_cell_2008}
For instance, Venet et al.\ have shown that, in two reference breast cancer
cohorts, most published signatures are significantly prognostic, yet no more
prognostic than random sets of genes.\cite{venet_most_2011}
% Here, we present an experimental design aiming at quantifying and characterizing
% the pervasiveness of prognostic signals in human cancer transcriptomes.
Here, we provide a global assessment of the pervasiveness of prognostic signals
in human cancer transcriptomes.

The chief observation issued from our analysis is the wide heterogeneity of
prognostic contents measured across the examined cancer cohorts.  When estimated
by the fraction of single gene-markers associated with outcome, our assessment
of transcriptomic prognostic content ranged from 2\% to 60\%; whereas a fraction
of 4722 random multi-gene markers ranging from 0.1\% to 94\% was shown to
inform patient differential survival across investigated datasets.
Interestingly, in most datasets, the fraction of randomized signatures
associated with outcome was only marginally inferior to the fraction of
biologically motivated prognostic MSigDB~\smallcaps{c2} signatures (median
difference: 2\%; maximum difference: 17\%).  In 37 out of the 114 analysed
datasets, the fraction of prognostic random signatures was higher than the
fraction of biologically motivated ones.

We have shown that this heterogeneity is not the result of biased survival times
constructs---the dependent variables of our fitted models.  Our estimate of
prognostic content is, however, notoriously sensitive to sampling variance, an
observation consistent with previous reports.\cite{ein-dor_outcome_2005} This
variance may be accounted for in part by a range of dataset-specific demographic
factors---such as duration of follow-up times or cohort size---, as well as by
cancer-specific biological covariates.  Furthermore, in at least one dataset
(\smallcaps{GSE9893}, single-gene marker prognostic content: 60\%), we have
identified a critical normalization artifact responsible for an artificial
over-estimation of the prognostic fraction.  Upon re-normalization of the
original arrays, we have re-evaluated our estimate of prognostic content down to
19\%; conversely, a metric of signal-to-noise ratio,\cite{venet_measure_2012}
surged from 15\% to 34\% (a score now within the inter-quartile range of the 114
datasets analysed).  This precedent suggests that artefactual spurious
correlations in expression matrices may account for significant portions of
prognostic content in our observations.

The scope of this analysis remains strictly epistemological.  While we have
observed that, in the thirteen breast cancer datasets with \smallcaps{os}
survival data, the single-features with highest scores of association with
\smallcaps{os} are largely overlapping (excluding \smallcaps{GSE9893}), the
validation of cancer-specific, biologically pertinent molecular markers from
microarray data is still a delicate task.  In any event, a more stringent
formulation of experimental controls is essential to the validation of the
predictive ability of biologically motivated
biomarkers.\cite{beck_significance_2013} Feature transformation techniques might
further be required to address spurious structures of correlation and assist to
the dissection of additional biological components in genomic prognostic
signals.

Taken together, these observations attest for the pan-transcriptomic nature of
prognostic signals in cancer expression profiles---as quantified by association
with outcome of single- and multi-gene markers with a Cox proportional hazards
model, at logrank $p<0.05$.  This illation is not cancer- or platform-specific;
therefore, transcriptomic prognostic signals cannot be convened to infer
specific biological mechanisms driving cancer progression.  While these
prognostic signals remain independent, and thus complementary, to
clinicopathological parameters, their ubiquity explains why many different
studies have reported incongruent outcome-specific predictive
signatures.\cite{gevaert_prediction_2009,chibon_cancer_2013} In addition, the
landscape of prognostic signals uncovered by our analysis may account for the
complexity of some challenges in clinical oncology---including the difficulty in
finding stable molecular predictors of response to specific systemic treatments
in breast cancer, for instance.\cite{reis-filho_gene_2011} We envisage the
phenomena here reported to be of significance for the interpretation of global
expression patterns of neoplastic samples; and to have a bearing in the
development of next generation genomic predictors.

% challenging the hypothesis that prognostic signatures reported in the literature
% are specific conveyors of biological signals pertaining to cancer progression.

\clearpage

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Discussion
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% Overall, the manner these re-sampling experiments impact the fluctuating of
% prognostic fractions in a reference breast cancer cohort is not surprising,
% and speaks for the nature

% Re-sampling strategy

% To discuss:
% -heterogeneity
% -preditor (single-genes, multi-gene markers)
% -predicted variable (survival times)
% -methodology employed to estimate outcome
% -GSE9893 (data not shown)

% Essentially, our analysis provides the clues to two interpret two
% puzzling/pressing questions regarding the survival outcome literature: why so
% many distinct signatures are associated with outcome in different platforms
% and why genomic analysis cannot provide a unique biomarker of response to
% treatment.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Thousands of studies have interpreted the correlation with cancer outcome of a
% transcriptional marker associated with a biological process as suggestive that
% this process contributes to cancer progression. This interpretation assumes
% that the correlation of the marker is specific, i.e. does not apply to all
% markers, a possibility uncontrolled for in most studies. Here we perform this
% control by quantifying the probability that random single-genes and
% multi-genes markers are prognostic in 114 datasets spanning human cancers
% afflicting 22 organ systems. We found that >5\% of the single- and multi-genes
% markers are associated with outcome at p<0.05 (log-rank test) in 73 and 83 of
% the datasets, respectively. Demographics, but also technical details of the
% studies and sampling effects modulate the probability of observing an
% association by chance. Thus, no biological conclusion can be drawn from the
% association at p<0.05 of a marker with cancer outcome.

% The vast majority of mechanistic cancer studies are performed in animals and/or
% in vitro for ethical reasons. Proving that a mechanism dissected in these
% experimental systems contributes to cancer progression in human requires
% clinical trails, which are costly and take years to perform. Therefore,
% researchers have relied on weaker correlative evidence to back the relevance of
% their findings to human diseases. One such evidence is the statistically
% significant association of a molecular maker of the mechanism under
% investigation with disease outcome in human. It can be established with
% non-interventional clinical studies. It has been extensively used in the past
% decades and has gained more popularity recently with the availability of web
% servers13providing, for free, association statistics for any single- or
% multi-gene markers and cancer outcome on the basis of published cancer
% transcriptome databases.

% To have any value, however, the reported association must be specific of the
% marker at hand and not reflect a global property of the transcriptome. Most
% studies estimate the association with a Cox analysis and a log-rank test and
% display it with a Kaplan-Meier curve. None of these controls for the possibility
% that a large fraction of transcriptome could be associated outcome and that
% therefore the prognostic effect is non-specific. Disturbingly, this possibility
% has been proven true in bladder4 and breast57 cancers. In particular,
% we have shown7 that more than half of the transcriptome is correlated with
% proliferation in two breast cancer cohorts and that 1 in 4 single gene and 9 in
% 10 random signatures of >100 genes were associated with overall survival at
% p<0.05. As a consequence, most published signatures were found to be
% significantly prognostic, but not more prognostic than random sets of genes.

% To investigate whether the pervasive association of the transcriptome with
% outcome extends to other cohorts and other types of cancers we compiled 114
% published gene-expression datasets with patient follow-up from the GEO8 and
% InSilicoDB9 databases. These included human cancers from 22 organ systems,
% and a wide range of transcriptomic platforms (Fig. 1; Supplementary Table
% S1). We computed for each dataset the fraction of genes associated with outcome
% at log-rank p<0.05 (Fig. 1, Supplementary Methods). This quantity may be viewed
% as the probability of observing a significantly prognostic single gene marker by
% chance alone. The median fraction was 9\%. It was higher than 5\% for 95 (83\%)
% datasets. To take an extreme example, an investigator measuring the association
% of a gene with outcome in the KIRC datasets (kidney cancer) has a 50\% chance to
% obtain a positive result, i.e. a value far above the canonical 5\% significance
% threshold. To investigate recent multi-gene approaches, we ran a similar
% calculation for 4722 signatures of the MSigDB database10 (Fig. 1; details in
% Supplementary Methods). The prognostic fraction was larger for signatures than
% for single genes, with a median of 12\%. It was larger than 5\% for 74\% of the
% datasets, larger that 20\% for 41 (36\%) and larger that 50\% for 13 (11\%). To
% control for possible bias of MSigDB toward oncology-related signatures, we reran
% the exact same calculation, but replacing each signature by a similarly sized
% set of randomly selected genes. The overall qualitative result is unaltered
% (Fig. 1).

% Intriguingly, the prognostic transcriptome fraction was extremely
% heterogeneous within datasets related to the same organ system. For example,
% it ranged from 4 to 37\% among the breast cancer datasets. To probe the
% contributions of the datasets technical and demographic specifics to this
% variability, we reran the single-gene analysis on subsets of the 1972
% transcriptome profiles available from the METABRIC11 study. First,
% repeated random selection of profiles reveals that sampling effects are
% substantial (prognostic fraction CI=6-17\%) with 500 patients (Fig. 2A), a
% size in par with the largest studies. Second, the prognostic fraction, as
% expected, depends on sample size (Fig. 2B). Interestingly, it shows no sign of
% inflection as N reaches 1750, by far the largest sample size in the
% field. Third, we investigated the impact of follow-up time by artificially
% truncating follow-up data (Fig. 2C; Supplementary Methods). Shorter and longer
% maximum follow-up times yielded lower prognostic fractions, consistent with
% the fact that incomplete data, but also late events12, are not predictable
% from primary tumor transcriptomes. Fourth, increasing the fraction of ER+
% patients up to 50\% increases the prognostic fraction, but has no effect
% beyond 50\% (Fig. 2D). This is compatible with the notion that transcriptional
% marker are more prognostic within the ER+ group and that the ER+ patients
% have, on average, a better prognostic than ER- patients.13 Fifth,
% increasing the fraction of node positive patients, which is correlated with ER
% status, also increases the prognostic fraction (Fig. 2E). Sixth, lower
% cellularity is unexpectedly associated with a higher prognostic fraction
% (Fig. 2F). This may reflect the role of the microenvironment in breast cancer
% progression and the massive impact of cell types proportions on the
% transcriptomes of bulk tissues.14 Finally, unique dataset-specific
% processing details also play a role. For example, dataset GSE9893 has one of
% the highest prognostic fractions (Fig. 1), but surprisingly its
% signal-to-noise metrics15 is very low (Supplementary Fig. S1). Examination
% of the data reveals that normalization was performed in two batches and
% induced massive spurious correlations between expression and outcome
% (Supplementary Fig. S1). Accordingly, a proper single-batch normalization of
% the raw expression data restores the signal-to-noise metrics to normal and
% decreases the prognostic fraction from 59\% to 19\% (Supplementary Fig. S1).

% We have shown the fraction of prognostic single- and multi-gene markers is
% greater that 5\% in the vast majority of the datasets. Furthermore, the
% probability of a significant single- or multi-genes marker association with
% outcome depends on cohorts demographics, but also to a large extent on
% technical factors that include sampling effects, cohort size, patient
% follow-up protocols, cellularity standards, protocol randomization and
% possibly other factors not addressed here. These findings call for the
% reappraisal of previous study resting on the associations of markers with
% outcome to support relevance to human cancer, including low-throughput
% PCR-based studies. They also call for study-specific controls akin to those
% presented Fig. 1 in future studies.

% A marker does not need to convey interesting biological information
% research-wise in order to be useful in the clinic. Therefore, our results have
% no bearing on the clinical utility of published marker associations.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\clearpage

% \subsection{Microarray data interpretation and analysis}
% \label{sec:discussion-interpretation-microarrays}
% Wednesday 04Mar2015

% End up with a wink to underlying models for cancer progression.  In other
% words, interpretation of microarray data is dependent on the underlying
% biological model for the system under study.

\clearpage

%%% Local Variables:
%%% TeX-engine: xetex
%%% mode: latex
%%% TeX-master: "../../thesis"
%%% End:
