\section{Microarray technology}
\label{sec:microarray-methods}

Microarray technology relies on the non-covalent, sequence-specific interaction
between complementary strands of nucleic acids\cite{watson_molecular_1953} to
detect and quantify specific populations of m\smallcaps{rna} in a solution.  A
microarray chip consists of a universe of oligonucleotide probes attached to a
substrate through covalent bonds.  Each such probe is synthesized to
specifically match a unique messenger \smallcaps{rna} molecule.  When the chip
is exposed to a solution of fluorecently labeled m\smallcaps{rna}s, only those
that hybridize with their respective probes will be retained upon washing off
non-specific bonding sequences.  This allows for the quantification of the
fluorescent signals emitted when the chip is scanned with a laser beam of a
specific wavelength.  The measured signals relay the relative quantity of each
m\smallcaps{rna} molecule assayed by the microarray, as each spot has a known
position on the chip (Figure~\ref{fig:microarray-economist}).

% Oligonucleotide microarrays often carry control probes designed to hybridize
% with RNA spike-ins. The degree of hybridization between the spike-ins and the
% control probes is used to normalize the hybridization measurements for the
% target probes.

\begin{marginfigure}%
  \begin{center}
    % \includegraphics[width=9cm]{microarrays-economist.jpg}
    \includegraphics{microarrays-economist-17Feb2015.pdf}
    \caption[Schematic representation of how microarrays work]{A schematic
      representation of how microarrays work.  \textbf{1.}~Microarrays rely on a
      fundamental property of nucleic acids, the monomeric units that polymerize
      into \smallcaps{dna} or \smallcaps{rna} strands.  Adenine (\smallcaps{A})
      are complementary to thymine (\smallcaps{t}), and cytosine (\smallcaps{c})
      are complementary to guanine (\smallcaps{g}).  Just one incorrect base can
      prevent two strands from binding.  \textbf{2.}~A microarray typically
      contains thousands of squares, or spots.  Each spot anchors many copies of
      a particular sequence of single-stranded \smallcaps{dna}, corresponding to
      a particular gene.  \textbf{3.}~Messenger \smallcaps{rna} fragments
      extracted from a tissue and labeled with different fluorescent dyes are
      washed over the microarray and hybridize with \smallcaps{dna} strands with
      the complementary sequence.  \textbf{4.}~The dyes are illuminated using
      fluorescent light.  It is then possible to show which \smallcaps{rna}
      fragments were retained in which spots---and hence which genes were being
      expressed in the tissue from which the \smallcaps{rna} was extracted.
      Source: \emph{The Economist; Affymetrix}.}
    \label{fig:microarray-economist}%
  \end{center}
\end{marginfigure}

\section{Microarray data preprocessing}
\label{sec:microarray-methods-data-preprocessing}

By virtue of their design, microarrays allow for the monitoring of expression
levels for thousands of gene transcription products simultaneously.  Microarray
expression data are thus characterized by high dimensionality and noisiness.
This prompts the need for preprocessing methods aiming at removing systematic
biases in expression measurements, introduced during
experimentation.\cite{shakya_comparison_2010}

The goal of microarray data preprocessing is to convert raw imaging data into
meaningful biological data and to enable comparison of results obtained from
different arrays.  It comprises three steps: (\emph{a})~the transformation of
image data into intensity values; (\emph{b})~the assessment of array quality;
and (\emph{c})~the removing of technical biases (through background adjustment,
normalization and feature filtering and summarization).

The digital imaging of fluorescence signals is typically performed by
proprietary software designed by the microarray manufacturer.  These software
packages assign coordinates to each spot in the array, quantify signal intensity
and uniformity of each spot, and compare their signal intensity relative to
background.

Quality control is performed by visual inspection of imaging data from the
scanner or the platform software, with special attention given to washing
artifacts, odd or missing spots, and array uniformity.  Aberrant chips may have
to be discarded at this stage.

Chips meeting quality standards then undergo background adjustment and
normalization.  Normalization methods aim to compensate for procedural biases
that are independent from biological signal.  Early approaches for microarray
normalization were based on the assumption that most genes, and in particular
so-called housekeeping genes,\footnote{\emph{Housekeeping} genes are genes
  defined as participating in basic, thus universal, cellular processes.}
should have similar expression levels across samples.  Housekeeping genes have
since been shown to vary in expression by 30\% or more across healthy samples,
and even more in tumour samples.\cite{lee_control_2002,eisenberg_human_2003}
Since then, data-driven normalization approaches were developed, such as median
correction,\cite{cho_genome-wide_1998,selinger_rna_2000} variance stabilizing
transformation,\cite{durbin_variance-stabilizing_2002} locally weighted linear
regression\cite{yang_normalization_2002} and spline based
methods.\cite{workman_new_2002} The underlying principle of these corrective
procedures is that most of the observed variation in expression values is due to
technical biases rather than to biological sources.\cite{hicks_when_2014}

Normalization strategies for double-channel microarrays (spotted oligonucleotide
or c\smallcaps{dna} arrays\cite{schena_quantitative_1995}) are different from
those for single-channel microarrays (\emph{in situ} synthesized high density
oligonucleotide arrays,\cite{lockhart_expression_1996} such as \emph{Affymetrix
  GeneChip}).  \emph{Affymetrix GeneChip} arrays use multiple probes per gene
and a single-colour detection system, as one sample is hybridized per chip.
Spotted oligonucleotide or c\smallcaps{dna} arrays use one probe per gene and a
two-colour scheme, where two different samples are hybridized on the same array.
Consequently, single-channel arrays measure the overall abundance of a probe
sequence in a target sample, whereas c\smallcaps{dna} arrays measure the
relative abundance of a probe sequence in two target samples.
% In other words, the expression measures for single-channel arrays are absolute
% (log) intensities, whereas they represent (log) ratios of intensities for
% c\smallcaps{dna} arrays.\footnote{In many cases, one of the samples in a
% c\smallcaps{DNA} array hybridization is a common reference used across
% multiple slides and whose sole purpose is to provide a baseline for direct
% comparison of expression measures between arrays.}
As a result, normalization of single-channel microarrays is performed at the
``between-array'' level, whereas normalization of double-channel microarrays is
conducted at the ``within-array'' level.\cite{do_normalization_2006}

For double-channel arrays chips, normalization methods commonly seek to remove
biases within each array with local regression algorithms.  Terry Speed's lab,
at Berkeley, identified an intensity-dependent dye bias concerning
c\smallcaps{dna} arrays.  In these arrays, the $\log_2(ratio)$ shows a
systematic dependence on intensity, characterized by a deviation from zero for
low-intensity spots.  Frequently, under-expressed genes appear up-regulated in
the red channel ($R$), and moderately expressed genes appear up-regulated in the
green channel ($G$).
% This artifact is the result of a ``quenching'' effect, whereby dye molecules
% in close proximity tend to re-absorb light from each other, hence diminishing
% the signal.  The amount of re-absorption is a function of the template
% concentration and differs for the two dyes.
This effect can be visualized by plotting the measured
$\log_2(\frac{R_{i}}{G_{i}})$ for each feature in the array as a function of the
$\log_2(R_{i}G_{i})$ product intensities.  This ratio-intensity plot is termed
\smallcaps{ma} plot.\footnote[][-4.5cm]{The name of the plot comes from
  ``minus'' and ``add'', respectively the ratio and product in the logarithmic
  scale.}  This technical bias may be corrected by fitting a locally weighted
regression known as \emph{lowess} smoothing
(Figure~\ref{fig:ma-plot}).\cite[-4.2cm]{yang_normalization_2001} More specific
sources of technical bias, including spatially-dependent bias resulting from the
print tips used in the manufacturing process of the array, may also be addressed
by a \emph{lowess}-based, within group normalization.

\begin{marginfigure}[-4.2cm]%
  \begin{center}
    \includegraphics{lowess-normalization-23Feb2013.pdf}
    \caption[\emph{Lowess} normalization]{Example of \emph{lowess}
      normalization.  \textbf{A:}~\smallcaps{ma} plot showing colour dye
      dependent bias.  \textbf{B:}~\smallcaps{ma} plot after correction with
      \emph{lowess} normalization (\citealp{yang_normalization_2002}).}
    \label{fig:ma-plot}%
  \end{center}
\end{marginfigure}

\emph{Affymetrix GeneChip} are the reference arrays in the single-channel class,
and the platform of choice for the development of between-array normalization
methods.  \emph{Affymetrix} chips consist of several tens of thousands
probe-sets.  A probe-set is a collection of probe pairs designed to interrogate
a specific sequence and contains \numrange{11}{20} probe pairs of 25-mer
oligonucleotides each.  Each probe pair consists of a perfect match probe
(\smallcaps{pm}) and a mismatch probe (\smallcaps{mm}).  The \smallcaps{mm}
probe differs from the \smallcaps{pm} probe by a single substitution at the
center base position, conceived to disturb the binding of the target gene
transcript.  This design allows for the quantification of background and
nonspecific hybridization effects.

Different between-array normalization methods have been proposed in the
literature.  The \smallcaps{mas5} algorithm\cite{hubbell_robust_2002} normalizes
the value of summarized probe-sets by linear scaling based on a reference array.
The \smallcaps{rma} (robust multi-array average)
algorithm\cite{irizarry_exploration_2003} normalizes the value of each probe by
quantile normalization\footnote{Quantile normalization is a global adjustment
  method that assumes the statistical distribution of expression values of each
  sample is the same.  Normalization is achieved by imposing the same
  distribution to all samples, using an average distribution as reference.  The
  average distribution is estimated from the average of each quantile across
  samples.} in multiple arrays.  The \smallcaps{gcrma}
algorithm\cite{wu_model-based_2004} applies the same normalization and
summarization methods as \smallcaps{rma}, but uses probe sequence information to
estimate and correct for probe affinity to non-specific binding.  More recently,
the f\smallcaps{rma} (frozen \smallcaps{rma}) algorithm\cite{mccall_frozen_2010}
leverages pre-computed estimates of probe-specific effects and variance from
public microarray databases to, in concert with the information from new arrays,
normalize and summarize the data.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% http://www.people.vcu.edu/~mreimers/OGMDA/normalize.expression.html
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% The first ``\mbox{across-array}'' normalization methods relied on the
% assumption that most genes, and in particular ``housekeeping''
% genes\footnote{\emph{Housekeeping} genes are genes recognized as participating
% in basic, thus universal, cellular processes.}, should not be expected to be
% differentially expressed across samples.  However, housekeeping genes have
% since been shown to vary in expression by 30\% or more across healthy samples,
% and even more in tumour samples.\cite{lee_control_2002} This lead to new
% approaches to identify stable patterns of expression around which to pin
% global levels of expression.  Yet, these approaches have been shown to suffer
% from significant levels of cross-hybridization between \mbox{25-mers} in
% \emph{Affymetrix} chips.

% Terry Speed's lab, at Berkeley, identified an important intensity-dependent
% dye bias regarding double channel microarrays, and introduced a popular method
% for adjusting it.  In these arrays, the $log_{2} (ratio)$ shows a systematic
% dependence on intensity, characterized by a deviation from zero for
% low-intensity spots.  Typically, under-expressed genes appear up-regulated in
% the red channel ($R$) and moderately expressed genes appear up-regulated in
% the green channel ($G$).  This artifact is the result of a ``quenching''
% effect, whereby dye molecules in close proximity tend to re-absorb light from
% each other, hence diminishing the signal.  The amount of re-absorption is a
% function of the template concentration and differs for the two dyes.  This
% effect can be visualized by plotting the measured
% $log_{2}(\frac{R_{i}}{G_{i}})$ for each feature in the array as a function of
% the $log_{2}(R_{i}G_{i})$ product intensities.  This ratio-intensity plot is
% termed \smallcaps{ma} plot.\footnote{The name of the plot comes from ``minus''
% and ``add'', or ratio and product in the logarithmic scale.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% Among these are methods designed to normalize single chips, known as
% ``within-arrays'' methods, such as quantile normalization, or \smallcaps{mas5}
% for \emph{Affymetrix} chips.

Several comparison articles on these and other normalization methods were
produced in the specialized
literature,\cite{ploner_correlation_2005,bolstad_comparison_2003,harr_comparison_2006}
and a more detailed technical review on the computational implementation of
these algorithms can be found in the context of the Bionconductor
project.\footnote{The Bioconductor project is an open source and open
  development software project for the analysis and comprehension of genomic
  data.  It is rooted in the open source statistical computing environment
  \textsf{R} (\citealp{gentleman_bioinformatics_2006}).}

\medskip

The normalized microarray data for $p$ genes and $n$ biological samples are
denoted by $X_{n \times p}$ such that $x_{ij}$ represents the expression of gene
$j$ of sample $i$.

\section{Microarray data analysis}
\label{sec:microarray-methods-data-analysis}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% http://discover.nci.nih.gov/microarrayAnalysis/Exploratory.Analysis.jsp
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

In cancer research, analysis of genomic experiments performed with
\smallcaps{dna} microarray data may address a variety of tasks, including
finding gene associations with particular phenotypes, tumour class discovery or
tumour class prediction.  The work contributed in this thesis concerns:
(\emph{a}) the class prediction of tumour types based on gene expression
profiles; and (\emph{b}) the modeling of association of gene expression profiles
with particular outcomes (e.g., death of a patient or relapse of disease), using
survival analysis.  Both these problems are examples of application of
supervised learning techniques to the analysis of microarray data.  In machine
learning,\footnote{Machine learning is a branch of computational science whose
  purpose is to implement algorithms capable to infer models from training data.
  Models derived from machine learning methods are then expected to make
  predictions or to inform decisions on testing data.} supervised learning
methods seek to infer a function from labeled training data.  Conversely,
unsupervised learning methods aim to find intrinsic structure from unlabeled
data.\cite{webb_statistical_2003} Unsupervised learning is thus suited to
uncover coherent genomic signals from microarray data, whereas supervised
learning can be used to find associations between genomic signals and phenotypic
classes of samples.

Due to the high dimensionality of microarray data, it is often necessary to use
methods for dimensionality reduction.  These include \emph{feature
  transformation} methods and \emph{feature selection}
methods.\cite{haibe-kains_identification_2009}

Feature transformation is an unsupervised approach that consists in reducing the
feature space of a microarray gene expression matrix, such that new features
retain biological pertinence, maximum information and generalizability to
similar experiments:

\begin{equation}
  \label{eq:feature-transformation}
  X_{n \times p} \to X'_{n \times p'} : p \gg p'
\end{equation}

Examples of microarray feature transformation techniques include principal
component analysis and clustering methods.

Feature selection techniques, on the other hand, are supervised approaches to
reduce data dimensionality.  In this work, particular attention will be given to
gene expression signatures, an example of microarray feature selection.  Gene
expression signatures are collections of genes sharing a combined expression
pattern associated with a given phenotype.  The range of biological phenotypes
confining the characterization of gene expression signatures may include the
specification of tumour classes,\cite{perou_molecular_2000} the modulation of
signaling pathways,\cite{itadani_can_2008} or the definition of distinct
clinical outcomes.\cite{vant_veer_gene_2002}

% metagenes. how are they derived. correlations with gene expression. they can
% behave as surrogate markers of biological processes, states or responses and
% be used as classifiers.  Metagenes can also be also non-biologically
% motivated, and thus be defined randomly.

% a word about meta-data resources and bioconductor tools.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% abstract of chapter seven of the book
%% Bioinformatics and Computational Biology Solutions Using R and Bioconductor
%% Meta-data Resources and Tools in Bioconductor
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% Closing the gap between knowledge of sequence and knowledge of function
% requires aggressive, integrative use of biological research databases of many
% different types.  For greatest effectiveness, analysis processes and
% interpretation of analytic results must be guided using relevant knowledge
% about the systems under investigation.  However, this knowledge is often
% widely scattered and encoded in a variety of formats.  In this section, we
% consider some of the different sources of biological information as well as
% the software tools that can be used to access these data and to integrate them
% into an analysis.  Bioconductor provides tools for creating, distributing, and
% accessing annotation resources in ways that have been found effective in work-
% flows for statistical analysis of microarray and other high-throughput assays.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\medskip

This section will proceed with a brief overview of tools for visualization of
genomic data, namely heatmaps and multidimensional scaling.  It will then
discuss the unsupervised learning tools used in this work, including principal
component analysis, clustering analysis and a summary on machine learning
methods.  Next, it will cover tools to evaluate the performance of classifiers,
including \smallcaps{roc} curves (for binary classifiers) and \smallcaps{gsea}
(an algorithm that uses microarray data to determine whether a gene expression
classifier is statistically associated with any of two phenotypic classes of
biological samples).  Finally, it will detail survival analysis, the branch of
supervised learning that seeks to explain the relationship between a number of
measured features (gene expression data) and the time duration until the
occurrence of a particular event (survival outcome).

\subsection{Visualization techniques}
\label{sec:methods-visualization}

\subsection{Principal component analysis}
\label{sec:methods-pc}

\subsection{Clustering analysis}
\label{sec:methods-clustering}

\subsection{Machine learning analysis}
\label{sec:methods-machine-learning}

\subsection{Receiver operating characteristic curves}
\label{sec:methods-roc}

\subsection{Gene set enrichment analysis}
\label{sec:methods-gsea}

\subsection{Survival analysis}
\label{sec:methods-survival-analysis}








%%% Local Variables:
%%% TeX-engine: xetex
%%% mode: latex
%%% TeX-master: "../../thesis"
%%% End:
